src/store/root-state.epics.ts

```diff
@@ -23,7 +23,7 @@ const rootStateResetEpic = (action$: Observable<Action>, state$: Observable<Root
     switchMap(keychainOptionsArray =>
       from(keychainOptionsArray).pipe(switchMap(options => Keychain.resetGenericPassword(options)))
     ),
-    switchMap(() => BeaconHandler.removeAllPermissions()),
+    switchMap(() => forkJoin([BeaconHandler.removeAllPermissions(), BeaconHandler.removeAllPeers(true)])),
```

Comment:
maybe we have to remove all peers at first, then remove all permissions? or it doesn't matter?

Reply:
I don't think it matters